logforwarder/src/apps/vespa-logforwarder-start/child-handler.cpp

```diff
@@ -15,13 +17,17 @@ ChildHandler::ChildHandler() : _childRunning(false) {}
 namespace {
 
 void
-runSplunk(const vespalib::string &prefix, const char *a1, const char *a2 = 0)
+runSplunk(const vespalib::string &prefix, std::vector<const char *> args)
 {
-    const char *argv[] = { 0, a1, a2, 0 };
     vespalib::string path = prefix + "/bin/splunk";
-    argv[0] = path.c_str();
-    LOG(debug, "starting splunk forwarder with command: '%s' '%s' '%s'",
-        argv[0], argv[1], argv[2]);
+    args.insert(args.begin(), path.c_str());
+    std::string dbg = "";
+    for (const char *arg : args) {
+        dbg.append(" '");
+        dbg.append(arg);
+        dbg.append("'");
+    }
+    LOG(debug, "starting splunk forwarder with command: %s", dbg.c_str());
```

Comment:
Could wrap this in `LOG_WOULD_LOG`, but this isn't exactly a hot inner loop so doesn't matter in practice